Source/Core/GroundPolylineGeometry.js

```diff
@@ -353,41 +353,42 @@ define([
         return result;
     }
 
+    function tangentDirection(target, origin, up, result) {
+        result = direction(target, origin, result);
+
+        // orthogonalize
+        result = Cartesian3.cross(result, up, result);
+        result = Cartesian3.normalize(result, result);
+        result = Cartesian3.cross(up, result, result);
+        return result;
+    }
+
     var toPreviousScratch = new Cartesian3();
     var toNextScratch = new Cartesian3();
     var forwardScratch = new Cartesian3();
-    var coplanarNormalScratch = new Cartesian3();
-    var coplanarPlaneScratch = new Plane(Cartesian3.UNIT_X, 0.0);
     var vertexUpScratch = new Cartesian3();
     var cosine90 = 0.0;
+    var cosine180 = -1.0;
```

Comment:
@lilleyse for context, `computeVertexMiterNormal` is meant to compute a normal pointing to the "right" side of a join between two line segments defined by 3 points, eventually this is used to compute normals on the start and end faces of the volumes around polyline segments.